tests/index.js

```diff
@@ -18,7 +19,24 @@ describe('demo app', function () {
 
   const removeStoredPreferences = function () {
     const productName = require('../package').productName
-    const userDataPath = path.join(process.env.HOME, 'Library', 'Application Support', productName)
+
+    let userDataPath = ''
+    switch (process.platform) {
+      case 'darwin':
+        userDataPath = path.join(process.env.HOME, 'Library', 'Application Support', productName)
+        break
+      case 'win32':
+        userDataPath = path.join('%APPDATA%', productName)
```

Comment:
Hmm, it seems `%APPDATA%` won't work. `process.env.HOME, 'AppData', 'Roaming'` might, but it depends a bit on where the app is loaded from right? Any suggestions?